### PR TITLE
phlog now doesn't ignite every tick

### PR DESCRIPTION
**Pull request by Iamgoofball**
it only ignites when applied

### CODE REVIEW

**Comment by WJohn** [comment]
Does this help reduce the 400 or so burn damage you experience even while sitting in space?

**Comment by Iamgoofball** [comment]
@WJohn reduce your fire stacks with water too

File: code/modules/reagents/Chemistry-Reagents/Pyrotechnic-Reagents.dm [hunk 1]
```diff
@@ -129,9 +129,13 @@
...
 
+
+/datum/reagent/phlogiston/reaction_mob(mob/living/M, method=TOUCH, volume)
+	M.IgniteMob()
+	..()
+
 /datum/reagent/phlogiston/on_mob_life(mob/living/M)
 	M.adjust_fire_stacks(1)
-	M.IgniteMob()
 	M.adjustFireLoss(0.2*M.fire_stacks)
```
Comment (phil235) [inline code/modules/reagents/Chemistry-Reagents/Pyrotechnic-Reagents.dm]: Apparently phlogiston has the same issue that I fixed for clf3: drenching yourself with water while having phlog in your system heals your burns (negative fire_stacks). Can you fix that?

Comment (Iamgoofball) [inline code/modules/reagents/Chemistry-Reagents/Pyrotechnic-Reagents.dm]: negative fire stacks heals burn damage? 

what the fuck

**Comment by Dorsisdwarf** [comment]
I don't like this, wasn't the whole point of phlogiston as opposed to, say, clf3 that you can't just roll around and put it out?

**Comment by Iamgoofball** [comment]
Well rrgo was threatening to go onto the server and spam it a bunch so players demand a nerf, so I figured I'd nerf it myself before someone else did a knee jerk reaction.

**Comment by Tokiko1** [comment]
It's very easy to counter already, just carry a fire extinguisher(which is avaible pretty much anywhere) and spray yourself, or stand under a shower. Please don't nerf it.

**Comment by Lumipharon** [comment]
It's basically useless if you nerf the reapplication thing.
You can make quite deadly shotgun darts thanks to it, but it's not that strong in syringes, and it's easily counterable.

**Comment by Ergovisavi** [comment]
>It's not that strong in syringes

It takes 1 minute and 15 seconds roughly for a 15 unit phlogiston syringe to go through your system. During that time, you have to either sit there extingushing yourself while sitting next to a water tank(And if your timing is off with the server ticks, your heat will still rise, killing you. Or if someone got rid of the water tanks, have fun looking for new extinguishers for a full minute), or sit under a shower, or expose yourself to the vacuum of space. It's also super, super easy to make.

It's too strong. Stopping the constant re-ignition does make it too weak, however... should be fixed in a different way.

**Comment by Lumipharon** [comment]
Or I could shot you with a syringe of 10 neurotox + 5 perflu and you're now stunned and muted for longer then it takes me to drag you into maint and throttle you to death.

Phlo is strong if you're unable to stop it in anyway.
In 60 seconds I can run to the showers and wrench one cold.

**Comment by Ergovisavi** [comment]
Are you implying that chemistry having access to a roundstart version of mute toxin is good design and that it's the standard we should hold things to?

**Comment by Lumipharon** [comment]
I brought it up before goonchem ever got merged, and nothing has ever been done about it.
So apparently, yes?

**Comment by Xhuis** [comment]
thumbsdown

**Comment by Iamgoofball** [comment]
@Ergovisavi alright so i'm gonna have to ask you politely:

stay out of chem please, I know you have this hardon for fucking with features, but I've finally gotten chemistry to a point where I don't have 5 different groups of people bitching about 5 different parts of the fucking system and "muh balance"

and even if I don't change it and you make the changes, guess who they're gonna bitch at?

me

doesn't matter if your name is on the PR, everyone's gonna throw a shitfit at me and bitch that I'm ruining chem again

chemistry is in a nice spot right now

useful for beginners, a challenge for masters

don't touch it

**Comment by Iamgoofball** [comment]
@Lumipharon funnily enough perf is a chemical people tell me is "useless", and the few who use it say it's pretty balanced since it makes someone take way more hits to kill as you can't rely on the bonus oxygen damage

**Comment by Lumipharon** [comment]
That's because people lack imagination.
The healing it does isn't usually relevent either, since if you're muting someone, that typically means you're going to drag them somewhere to murder them anyway.

**Comment by Iamgoofball** [comment]
Plus, if you can mute them than you could just as easily knock them out, which is the same plus they can't hear.

**Comment by Ergovisavi** [comment]
Perfluorodecalin muting them happens on tick 1. Which is infinitely better than 12 ticks with morphine, infinitely better than 10 ticks from chloral hydrate, 18 ticks on neurotoxin (drug), 10 ticks on sodium_thiopental, 22 ticks with sulfonal, or the various other chems that knock you out.

So no, you can't "just as easily knock them out".

**Comment by Dorsisdwarf** [comment]
People on basil don't demand a nerf, even though every round as a chemist I make a shitload of it and use it via spraybottles along with napalm on anything that so much as looks at me funny.

**Comment by Cheridan** [comment]
@Iamgoofball I don't know if "one guy threatened to abuse it egregiously" is very good grounds for a change. If you want to put it to a poll or whatever then go ahead, otherwise it doesn't sound like there's a lot of people asking for this.

**Comment by SconesC** [comment]
@Cheridan It's multiple people now and this should really be looked into

A bottle thrown on you because of glorious #10244 is death with no counterplay sans CALOMEL which isn't exactly super easy to get. Even the bodies burn. They burn to husks, processing the chem. It's dumb and this is a good change.

**Pull request closed without merging**